cmd/container.go

```diff
@@ -15,15 +15,14 @@
 package cmd
 
 import (
-	"github.com/spf13/cobra"
 	"github.com/openconfig/containerz/client"
+	"github.com/spf13/cobra"
 )
 
 var containerCmd = &cobra.Command{
 	Use:   "container",
 	Short: "General container operations",
 	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
-		RootCmd.PersistentPreRun(cmd, args)
```

Comment:
Intentional?

Reply:
Yes - these should have been removed. They cause a nil dereference.